services/util_service.proto

```diff
@@ -34,5 +34,5 @@ service UtilService {
   /**
    * Generates a pseudorandom number.
    */
-  rpc prng (Transaction) returns (TransactionResponse);
+  rpc utilPrng (Transaction) returns (TransactionResponse);
```

Comment:
This is actually not consistent with other services. Since we're already in the `UtilService` we don't need to specify `util` again. Instead it should just be the `(action)` in `(service name)(action)(TransactionBody)` like we do in crypto_service, etc. So should revert this change

Reply:
@steven-sheehy  `TokenService` has all transaction names ending with `Token` like `mintToken`, instead of `mint`. Similarly in other services as well.

Reply:
No service method has the service name first. Some of them have the Hedera entity it's acting upon after the action. This makes sense since the `TokenService.mintToken()` is minting a token, the `CryptoService.createAccount()` is creating an account, etc. The `UtilService.utilPrng()` is not prng'ing a util or util'ing a prng.

Reply:
😄 Fixed. Thanks !